modules/ROOT/pages/engine-architecture-overview.adoc

```diff
@@ -91,7 +88,7 @@ BPM services are related to BPM activities.
 |  |
 
 | Description:
-| Manage categories of processes. A category is a way to classify processes. Categories are handled manually by administrators in Bonita Portal and are visible to portal users.
+| Manage categories of processes. A category is a way to classify processes. Categories are handled manually by in the Bonita Administrator Application and are visible to all users.
```

Comment:
Categories are handled manually by in the Bonita => Categories are handled manually by administrators in the Bonita

Reply:
Added it